mobileapp/components/tile/TileRenderer.tsx

```diff
@@ -51,48 +51,53 @@ const COMPONENTS_FOR_SYMBOLS = {
   O: <View></View>,
 };
 
+type play = (coordinates: TileCoordinate) => void;
+
 type TileProps = {
   tile: Tile;
   coordinates: TileCoordinate;
+  play: play;
 };
 
-const TileRenderer = ({ tile, coordinates }: TileProps) => {
+const TileRenderer = ({ tile, coordinates, play }: TileProps) => {
   const PLAYER_COLORS = {
     white: "white",
     black: "black",
   };
 
   return (
     <View style={styles.container}>
-      <Hexagon
-        colorFill={`${COLOR_PALETTE[tile.color]}`}
-        opacity={tile.playedBy != null ? "0.5" : "1"}
-      />
-      {tile.playedBy && (
+      <Pressable onPress={() => play(coordinates)}>
```

Comment:
Should'nt we have a highlight on simple onPress and register action on onLongPress events ?
You could probably make a translucent hexagon appear on top of the default one to make this happen.

Reply:
Maybe we can leave it like this and improve the style after the feedback of the PO ?